File: programmers/난이도별/level01.두_정수_사이의_합/areumsheep.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int temp = 0;
        
        if(a>b) {
            temp = b;
            b = a;
            a = temp;
        }
        
        for(int i=a; i<=b; i++)
            answer += i;
        
        return answer;
    }
}

File: programmers/난이도별/level01.로또의_최고_순위와_최저_순위/areumsheep.js
function solution(lottos, win_nums) {
    var answer = [];
    let min = 0, max = 0;
    min = lottos.filter(num => win_nums.includes(num)).length;
    max = lottos.filter(num => num === 0).length + min;
    
    answer.push(max > 1 ? 7-(max) : 6);
    answer.push(min > 1 ? 7-(min) : 6);
    
    return answer;
}
